src/libraries/Common/src/Interop/Windows/Kernel32/Interop.STORAGE_READ_CAPACITY.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        // https://docs.microsoft.com/en-us/windows/win32/devio/storage-read-capacity
+        [StructLayout(LayoutKind.Sequential)]
+        internal unsafe struct StorageReadCapacity
```

Comment:
```suggestion
        internal unsafe struct STORAGE_READ_CAPACITY
```
The casing should match the Windows SDK casing